# Goal: Waypoints
geometry_msgs/Pose[] target_waypoints
---
# Outcome
uint32 node_count
geometry_msgs/Pose[] path_waypoints
---
# Feedback: Current pose
float32 dist_from_target
uint32 node_count
